readme.md

```diff
@@ -1,151 +1,132 @@
 
-#### Renovate Me extension
+### Renovate Docs
 
-Go to the Visual Studio Marketplace and install the [Renovate Me](https://marketplace.visualstudio.com/items?itemName=jyc.vsts-extensions-renovate-me) extension in your organization.
-From there you can create a pipeline with the `RenovateMe` task.
+- [Renovate Configuration](https://docs.renovatebot.com/configuration-options/)
+- [Mend Renovate Self-Hosted Docs](https://github.com/mend/renovate-ce-ee/tree/main/docs)
 
-> [!NOTE]
-> This extension is created and maintained personally by a Renovate developer/user so support requests relating to the extension itself cannot be answered directly in the main Renovate repository.
+### Comparisons
 
-#### Custom pipeline
+- [Different ways to run Renovate](https://www.mend.io/renovate/)
+- [Renovate vs Dependabot](https://docs.renovatebot.com/bot-comparison/)
 
-You can create a custom pipeline with a `yml` definition that triggers `npx renovate`.
-[More details on how to configure the pipeline](https://docs.renovatebot.com/modules/platform/azure/).
+## Get involved
 
-### Bitbucket Cloud/Server, Forgejo, Gitea, GitLab
+### Issues and Discussions
 
-For Bitbucket Cloud, Bitbucket Server, Forgejo, Gitea and GitLab, use our self-hosting option.
+Please open a Discussion to get help, suggest a new feature, or to report a bug.
+We only want maintainers to open Issues.
 
-## Configuration
+- [GitHub Discussions for Renovate](https://github.com/renovatebot/renovate/discussions)
 
-Go to our [documentation website](https://docs.renovatebot.com/) to learn how to configure Renovate.
-We have a [full list of configuration options](https://docs.renovatebot.com/configuration-options/).
+### Contributing
 
-To get help with your configuration, go to the [discussions tab in the Renovate repository](https://github.com/renovatebot/renovate/discussions) and open a new "config help" discussion post.
+To contribute to Renovate, or run a local copy, please read the contributing guidelines.
 
-## Self-Hosting
+- [Guidelines for Contributing](https://github.com/renovatebot/renovate/blob/main/.github/contributing.md)
+- Items that need contribution: [good first issues](https://github.com/renovatebot/renovate/contribute)
 
-To run your own instance of Renovate you have several options:
+### Contact and Social Media
 
-- Install the `renovate` CLI tool from npmjs, run it on a schedule (e.g. using `cron`)
-- Run the `renovate/renovate:full` Docker Hub image (same content/versions as the CLI tool), run it on a schedule
-- Run the `renovate/renovate:latest` Docker Hub image if you only use package managers that don't need third-party binaries (e.g. JavaScript, Docker, NuGet, pip)
+The Renovate project is proudly supported and actively maintained by Mend.io.
 
-[More details on the self-hosting development](https://github.com/renovatebot/renovate/blob/main/docs/usage/getting-started/running.md).
+- Contact [Mend.io](http://Mend.io) for commercial support questions.
```

Comment:
```suggestion
- Contact [Mend.io](https://www.mend.io/) for commercial support questions.
```

- Links should use HTTPS
- Links are lowercase

Please check if the new link is correct. 😉